### ISSUE TITLE
Create a new project bt importing a Taiga project JSON is broken (/project/new/import)  

### ISSUE THREAD

**Issue by andreashaerter** [taigaio/taiga-back#1322]
**Do you want to request a *feature* or report a *bug*?**

Bug

**What is the current behavior?**

Importing a project from an exported JSON file does not work.

**If the current behavior is a bug, please provide the steps to reproduce.**

1. Create fresh Taiga instance
2. Create a new project1 with one user story 
3. Export the project into a JSON file (note: projects from sample data will result in the same error but lager JSON, for sure. May be harder to debug)
4. Delete project one or switch to another taiga instance
5. Create a new via `/project/new/import` -> Provide the exported JSON -> Error (no matter how small the project is, sample attached: [admin-testprojectsource-9127c0550bc54dea9ae01bc36553cfcb.json.txt](https://github.com/taigaio/taiga-back/files/3278736/admin-testprojectsource-9127c0550bc54dea9ae01bc36553cfcb.json.txt)

**What is the expected behavior?**

Creating a project using the Import function results in a successful import.

**Is it happening in taiga.io or in your own instance?**

Own instance installed one day ago. Running Taiga on Python 3.6, Nginx, CentOS7. Checked out stable, `git describe --tag` reports 4.2.4 for backend and 4.2.5-stable for frontend.

**What browser/version are you using?**

* Firefox 67.0 (64-Bit) on Fedora Linux
* Google Chome 75.0.3770.80 (64-Bit) on Fedora Linux

**Are there any console errors *(Ctrl + F12)* in red?**

> POST https://taiga-test.example.com/api/v1/importer/load_dump 400 (Bad Request) app.js:25496

Sadly, no Grunicorn Errors appear in the log.

**Comment by alexhermida** [comment]
@andreashaerter we just imported without issues the attachment. 
Could you try to import manually the file with the `python manage.py load_dump file.json owner_email` command?

**Comment by andreashaerter** [comment]
@alexhermida Thank you for your example. The terminal gives a reasonable error message the WebUI does not show:

```
$ su - taiga
$ workon taiga
(taiga) [taiga@t... taiga-back]$ python manage.py load_dump ~/admin-testprojectsource-9127c0550bc54dea9ae01bc36553cfcb.json admin@admin.com
Trying import local.py settings...
ERROR: error importing memberships
{
    "memberships": [
        {
            "email": [
                "You email domain is not allowed"
            ]
        }
    ]
}
```

So the problem is: on a fresh setup, the predefined Admin-user has the email `admin@admin.com`. And I used the [officially reserved example domain](https://www.iana.org/domains/reserved) `example.com` as allowed email domain only:

```
$ grep -i "USER_EMAIL_ALLOWED_DOMAINS" ~/taiga-back/settings/local.py
USER_EMAIL_ALLOWED_DOMAINS = ["example.com"]
```
So it was **not a bug** but just

* unexpected behaviour for me (I did not expect this will limit existing users with non-listed domains to manage projects or memberships)
* a insufficient, generic error message in the UI

So thank you again for your feedback, pushed me into the right direction. :-)


PS: it would be preferable to **use `example.com` or `example.org` for sample-data** in the future as `admin.com` is a real, existing third party domain.
